### DISCUSSION TITLE
Pagination not on "List All Categories" view

### DISCUSSION THREAD

**Discussion by sjkelley** [joomla/joomla-cms#14393] [2. Cold Features]
In Joomla 3.6.5 there is no category pagination in the "View All Categories" menu item type. "Pagination" is set to "Show" and "# of Articles" is set in the menu item parameters.

Article pagination works correctly, but the list of categories is not paginated.

Pagination code needs to be added to /components/com_content/views/categories/tmpl/default_items.php

**Comment by brianteeman** [comment]
I can confirm that there are no option for this menu type to have a list limit on the number of categories displayed in the list.  (the options you refer to are for the content of the categories not for the list itself

**Comment by sjkelley** [comment]
Brian,

Yes, that's what I've determined too.  Are there any plans to add pagination support to the "List All Categories" view in the near future?  After doing some searching I can see people have been requesting this feature as far back as 2010.

If not, do you know of any patches, hacks or modified versions of:
 /components/com_content/views/categories/tmpl/default_items.php  that include pagination?

**Comment by huynhkhanhduc** [comment]
I fixed by using some trick.

Here are what I did in this file (/components/com_content/views/categories/tmpl/default_items.php)
<img width="848" alt="screen shot 2017-05-18 at 2 42 31 pm" src="https://cloud.githubusercontent.com/assets/2680362/26191491/4591a1ee-3bd8-11e7-93b8-17e17c3f5d7d.png">

**Comment by ghost** [comment]
@huynhkhanhduc can you make a PR (i didn't understand your Answer in other Thread)?

**Comment by ghost** [comment]
Status is set on "Needs Review".
